apps/web/vibes/soul/sections/sticky-sidebar-layout/index.tsx

```diff
@@ -7,9 +18,9 @@ import { clsx } from 'clsx';
  * ```css
  * :root {
  *   --section-max-width-medium: 768px;
- *   --section-max-width-lg: 1024px;
- *   --section-max-width-xl: 1280px;
- *   --section-max-width-2xl: 1536px;
+ *   --section-max-width-large: 1024px;
```

Comment:
Should we not match these names up with the names we use in `containerSize` and `sidebarSize`?